applet/src/menu.c

```diff
@@ -134,23 +134,49 @@ typedef struct menu_mem_base_type menu_mem_base_t ;
 
 extern menu_mem_base_t md380_menu_mem_base[];
 
-void create_menu_entry_rev(int menuid, const wchar_t * label , void * green_key, void  * red_key, int e, int f ,int enabled) 
+void create_menu_entry_rev(int menuid, const wchar_t * label , void * green_key, void  * red_key, int e, int f ,int item_count) 
```

Comment:
@aeickho  i ve experimented with '5' but no difference in timeout. '0' means a total hangup until poweroff with volume knob, then there is a screen refresh